INSTALLATION_GUIDE.md

```diff
@@ -0,0 +1,41 @@
+## Docker enviroment
+
+### Installation
+
+Docker allows you to run identical software on all platforms. In
+Windows, in particular, there are challenges ensuring all python
+packages are exactly the same as those available on other platforms.
+
+Windows and Macs have different installation processes. Regardless of machine, you will have to install 
+Docker and make an acccount. Please follow installation instructions on the docker website
+[here](https://docs.docker.com/install/) for how to complete this step. Docker Desktop is preferred over Docker Toolbox. 
```

Comment:
It's a good habit when possible to make links be meaningful for accessibility reasons (so `[on the docker website]` rather than `[here]`)

Reply:
done